packages/salesforcedx-vscode-core/src/commands/templates/forceFunctionCreate.ts

```diff
@@ -44,6 +47,16 @@ export class ForceFunctionCreateExecutor extends BaseTemplateCommand {
       .withLogName('force_create_function')
       .build();
   }
+
+  public runPostCommandTasks(outputDir: string) {
```

Comment:
The outputDir here could potentially different from the outputDir input by the user (`response.data.outputDir`), it's the directory of the file that VS Code is going to open. Could we add a comment to clarify?

Reply:
I've changed it to targetDir to avoid that confusion.